integration-tests/docker/druid.sh

```diff
@@ -73,4 +76,20 @@ setupConfig() {
       var=$(echo "$evar" | sed -e 's?^\([^=]*\)=.*?\1?g' -e 's?_?.?g')
       setKey $DRUID_SERVICE "$var" "$val"
   done
-}
\ No newline at end of file
+}
+
+setupData()
```

Comment:
This will work for how our travis CI is setup but if user try running manually locally with...
1) -DexcludedGroups=not-query
2) without -Dgroups and without DexcludedGroups
then the data won't be setup

Reply:
Added comments indicating the above.

Reply:
Ah, one more thing I forgot about, please update the docs in https://github.com/apache/druid/blob/master/integration-tests/README.md to include instructions for how to supply credentials and run these tests

Reply:
Done